docs/hosting/configuration/configuration-methods.md

```diff
@@ -117,119 +118,4 @@ The following environment variables support file input:
 
 ## Examples
```

Comment:
Please remove the entire examples section and the link - it should be obvious enough in the menu